networkx/algorithms/euler.py

```diff
@@ -382,7 +382,10 @@ def eulerian_path(G, source=None, keys=False):
 
 @not_implemented_for("directed")
 def eulerize(G):
-    """Transforms a graph into an Eulerian graph
+    """Transforms a graph into an Eulerian graph.
+
+    If `G` is Eulerian the result is `G`, otherwise the result is a smallest
```

Comment:
Should this perhaps mention that the result is always a multigraph, i.e. something like "If `G` is Eulerian the result is `G` as a MultiGraph..."?

Reply:
Sounds good! Changed to "... G as a `MultiGraph`". Not sure if Pascal case is better here, since later multigraph is spelled with lower case letters, or it does not matter.